tensorboard/webapp/metrics/views/card_renderer/image_card_container.ts

```diff
@@ -57,6 +57,8 @@ import {CardRenderer} from '../metrics_view_types';
 import {getTagDisplayName} from '../utils';
 import {maybeClipSelectedTime, ViewSelectedTime} from './utils';
 
+const DISTANCE_RATIO = 0.1;
```

Comment:
I'm fine with this choice and I like that the code is written in a way that lets us play with ratio. However, I'm curious: Why not something closer to 0.5?